leetcode/src/16.c

```diff
@@ -0,0 +1,30 @@
+#include <stdlib.h> // for qsort()
+
+int cmp(const void* a, const void* b) {
+    return *(int*)a - *(int*)b;
```

Comment:
Actually, comparing two numbers by subtraction is bad practice, the sort will break if the result overflows. [Source](https://stackoverflow.com/a/27284248/918959)
```suggestion
   const int *A = a, *B = b;
   return (*A > *B) - (*A < *B);
```

Reply:
Just updated it. For this problem though, overflow could not happen as it is specified that abs(nums[i]) <= 1000.

Reply:
> Just updated it. For this problem though, overflow could not happen as it is specified that abs(nums[i]) <= 1000.

Yeah, I know that, it's actually one thing I really disliked about those leetcode problems. IRL people input all kinds of stuff into programs, and programmers must have contingency plans for everything or there's going to be a problem. These exercises might be good for familiarizing yourself with the language or various algorithms, but they teach you nothing about best practices that make code more efficient, more maintainable and less bug prone. Competitive programming/Interview questions and actual software engineering are really _very_ different things. By the way, sync your fork and enable actions for it, and here's [some nice reading material](https://sekrit.de/webdocs/c/beginners-guide-away-from-scanf.html)